portiaui/app/components/list-item-link-crawling.js

```diff
@@ -33,7 +33,11 @@ export default Ember.Component.extend({
 
     actions: {
         saveSpider() {
-            this.get('spider').save();
+            this.get('spider').save().then(() => {
+                if (this.get('linksToFollow') !== 'none') {
```

Comment:
my guess would be the user only expects to see the options when they want to set patterns.
in the other case it may be annoying espacialy if they switch to 'none' and then back to 'all'.
lets go with just showing it for patterns for now, and wait for feedback